sklearn/gaussian_process/_gpr.py

```diff
@@ -302,7 +302,7 @@ def predict(self, X, return_std=False, return_cov=False):
 
         Returns
         -------
-        y_mean : ndarray of shape (n_samples, [n_output_dims])
+        y_mean : ndarray of shape (n_samples, [n_targets])
```

Comment:
I do not think there was a reason to have the `[]`:

```suggestion
        y_mean : ndarray of shape (n_samples, n_targets)
```

Reply:
The brackets `[]` indicate that `y_mean` has shape either `(n_samples,)` or `(n_samples, n_targets)`. The shape of `y_mean` depends on the training data. There needs to be a way to convey this either-or situation in the documentation. Brackets are not necessarily the best way, but simply removing the brackets would be incorrect.

Reply:
We have not been using brackets like this for recent docstrings, can we change this to:

```suggestion
        y_mean : ndarray of shape (n_samples,) or (n_samples, n_targets)
```